app/screens/channel/channel_post_list/channel_post_list.js

```diff
@@ -128,10 +134,16 @@ export default class ChannelPostList extends PureComponent {
         } = this.props;
 
         const {
-            visiblePostIds
+            visiblePostIds,
+            loading
         } = this.state;
 
+        if (loading) {
+            return null;
```

Comment:
@enahum Should we maybe show the loading spinner here so that it's not blank? That'll still be much cheaper to render than the whole post list

Reply:
Yes that makes sense

Reply:
this would have been shown for a very little time, and this way it creates a better transition in my opinion

Reply:
in case of a channel switch it would not have been shown at all